app-backend/tile/src/main/scala/routes/MosaicRoutes.scala

```diff
@@ -41,33 +42,34 @@ object MosaicRoutes extends LazyLogging with KamonTraceDirectives {
     pathPrefix("export") {
       optionalHeaderValueByName("Accept") { acceptContentType =>
         traceName("tiles-export-request") {
-          parameter("bbox", "zoom".as[Int]?, "colorCorrect".as[Boolean] ? true) {
-            (bbox, zoom, colorCorrect) => get {
-              complete {
-                val mosaic = Mosaic.render(projectId, zoom, Option(bbox), colorCorrect)
-                val poly = Projected(Extent.fromString(bbox).toPolygon(), 4326)
-                  .reproject(LatLng, WebMercator)(3857)
-                val extent = poly.envelope
-                val future = acceptContentType match {
-                  case Some("image/tiff") => mosaic
-                      .map{ m => MultibandGeoTiff(m, extent, WebMercator) }
+          parameter("bbox", "zoom".as[Int] ?, "colorCorrect".as[Boolean] ? true) {
+            (bbox, zoom, colorCorrect) =>
+              get {
+                complete {
```

Comment:
Haha forget it, it's fine; removed my question as figured that everything is fine ^^'